### PR TITLE
rebuild 0.16

### PR DESCRIPTION
**Pull request by boldorider4**
* [x] Dependency for orange3 requiring docutils <0.17
* [x] Used a fork of the feedstock at 0.16 release
* [x] support py39..311 on all platforms
* [x] Bumped the build number to 2
* [x] License file, family and other fields have been made compliant
* [x] About and host section have been updated to comply with linter

### CODE REVIEW

File: abs.yaml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+upload_channels:
```
Comment (M-Waszkiewicz-Anaconda) [inline abs.yaml:1]: do no forget to remove this file

**Review by M-Waszkiewicz-Anaconda — APPROVED** [review]
LGTM

File: recipe/meta.yaml [hunk 2]
```diff
@@ -9,13 +9,14 @@ source:
   sha256: c2de3a60e9e7d07be26b7f2b00ca0309c207e06c100f9cc2a94931fc75a478fc
 
 build:
-  number: 0
-  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+  number: 2
```
Comment (M-Waszkiewicz-Anaconda) [inline recipe/meta.yaml:12]: number 1 ?

Comment (M-Waszkiewicz-Anaconda) [reply recipe/meta.yaml:12]: actually I think you messed up with branches, you should branch out from master not main 0.16 and your build number should be +1 of this one https://github.com/AnacondaRecipes/docutils-feedstock/blob/35c78f430afb2e780a84f9ef9929ad1de455b690/recipe/meta.yaml#L13

Comment (M-Waszkiewicz-Anaconda) [reply recipe/meta.yaml:12]: speaking with Jack actually it is good  as he need old version of this

**Review by M-Waszkiewicz-Anaconda — APPROVED** [review]
LGTM

**Review by skupr-anaconda — APPROVED** [review]

**Review by JeanChristopheMorinPerso — APPROVED** [review]

**Pull request merged**